Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/BlobTriggerCSharp.cs

```diff
@@ -3,23 +3,22 @@
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 
-namespace Company.Function
+namespace Company.Function;
+
+public class BlobTriggerCSharp
```

Comment:
Could also use primary constructors.

```suggestion
public class BlobTriggerCSharp(ILogger<BlobTriggerCSharp> logger)
```

Then don't even have the `_logger` field. You can access the ctor parameter anywhere in this class via `logger`.

Reply:
I'm eager to get to primary constructors too, but there was an open question around C#12/.NET 8 SDK checks. To date, we've largely kept the item templates agnostic to things like framework, with the exception of HTTP, which has a much more complicated handling since we can't rely on a restore. I'm not sure if the SDK version is available to the template engine without that, but it's a consideration we need to make since the templates can update independently. For now, omitting from this PR, since this is a clearer set of wins that we could deliver sooner.